frontend/src/pages/home/home.tsx

```diff
@@ -57,6 +61,13 @@ export default class HomePage extends React.Component<HomePageProps, HomePageSta
                     this.setState({loading: false});
                 })
         }
+        this.manoAloeService.getAllAnnouncements()
```

Comment:
Maybe add the loading part in this data fetching part as well, it does not make sense to leave it out